src/brsTypes/Callable.ts

```diff
@@ -57,7 +56,7 @@ export class StdlibArgument implements Argument {
     }
 
     /** A fake location exists only within the BRS runtime. */
-    static InternalRange = util.createRange(-1, -1, -1, -1);
+    static InternalRange = Range.create(-1, -1, -1, -1);
```

Comment:
This was to work around a ts-node mocha testing circular reference issue that only showed up during local debugging of BrighterScript.